tests/sqllogictests/suites/tpch_iceberg/utils.test

```diff
@@ -84,6 +86,8 @@ show databases;
 ----
 abc
 ef123
+information_schema
+system
```

Comment:
1. need a test to verfiy `system.tables` contains `lineitem` in iceberg catalog.
2. need a test to verfiy `system.columns` contains `l_quantity` in iceberg catalog.

Reply:
Test because the show tables will query from the current_catalog.system.tables.